### ISSUE TITLE
$connectedsessions | Disconnect-HPOVMgmt does not work

### ISSUE THREAD

**Issue by VincentBerger** [HewlettPackard/POSH-HPOneView#364]
### Expected Behavior
According to the help (example 3 in https://github.com/HewlettPackard/POSH-HPOneView/wiki/Disconnect-HPOVMgmt) Disconnect-HPOVMgmt should take pipeline input to disconnect multiple appliance connections at once.

### Actual Behavior
But it doesn't:

Required? | true
Position? | 0
Default value | [System.Collections.ArrayList]${Global:ConnectedSessions}.Clone()
**Accept pipeline input? | false**
Accept wildcard characters? | False

### Steps to reproduce

```PowerShell
$ConnectedSessions | Disconnect-HPOVMgmt -verbose
VERBOSE: [DISCONNECT-HPOVMGMT] Bound PS Parameters: 
Key     Value
---     -----
Verbose True 



Disconnect-HPOVMgmt : The input object cannot be bound to any parameters for the command either because the command does not take pipeline input or the input and its 
properties do not match any of the parameters that take pipeline input.
At line:1 char:22
+ $ConnectedSessions | Disconnect-HPOVMgmt -verbose
+                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (HPOneView.Appliance.Connection:PSObject) [Disconnect-HPOVMgmt], ParameterBindingException
    + FullyQualifiedErrorId : InputObjectNotBound,Disconnect-HPOVMgmt
 
Disconnect-HPOVMgmt : The input object cannot be bound to any parameters for the command either because the command does not take pipeline input or the input and its 
properties do not match any of the parameters that take pipeline input.
At line:1 char:22
+ $ConnectedSessions | Disconnect-HPOVMgmt -verbose
+                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (HPOneView.Appliance.Connection:PSObject) [Disconnect-HPOVMgmt], ParameterBindingException
    + FullyQualifiedErrorId : InputObjectNotBound,Disconnect-HPOVMgmt
 

ConnectionID Name              UserName      AuthLoginDomain Default
------------ ----              --------      --------------- -------
1            composer.etss.lab Administrator LOCAL           True   
2            192.168.1.51      Administrator LOCAL           False  

```

### Version Information
HPE OneView PowerShell Library Version (`Get-HPOVVersion` or `$PSLibraryVersion`): 4.0.1672.1789
HPE OneView Appliance Version (`Get-HPOVVersion -ApplianceVer`):  4.00.07.330056.00
Output from `$PSVersionTable` on your Windows Host:

```PowerShell
PSVersion                      5.1.16299.611                                                                                                                             
PSEdition                      Desktop                                                                                                                                   
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                                                                                                                   
BuildVersion                   10.0.16299.611                                                                                                                            
CLRVersion                     4.0.30319.42000                                                                                                                           
WSManStackVersion              3.0                                                                                                                                       
PSRemotingProtocolVersion      2.3                                                                                                                                       
SerializationVersion           1.1.0.1                                                                                                                                   
```

**Comment by ChrisLynchHPE** [comment]
This is a documentation issue.  Because the entire pipeline isn't visible during the Process block, you can't clone the object and iterate through the clone.  When you attempt to modify an array that is being enumerated (like PowerShell does via the Pipeline), an error will be generated saying `Collection was modified; enumeration operation may not execute.`

So I will update the user documentation removing the example.

I'll also look at adding a switch parameter that would logoff all sessions.

**Comment by ChrisLynchHPE** [comment]
I actually went back and looked at the code.  I'll be fixing this, as it should be supported.  This will be fixed in both the 4.00 and 4.10 library updates.

**Comment by ChrisLynchHPE** [comment]
This is fixed in [Release 4.10.1821.1567](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v4.10.1821.1567) and in [PowerShell Gallery](https://www.powershellgallery.com/packages/HPOneView.410/4.10.1821.1567), and also [Release 4.0.1821.1629](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v4.0.1821.1629) and in [PowerShell Gallery](https://www.powershellgallery.com/packages/HPOneView.400/4.0.1821.1629).

Please verify before this issue is closed.

**Comment by VincentBerger** [comment]
Verified, closing
